### PR TITLE
Bazel support

### PR DESCRIPTION
**Pull request by bowlofstew**
This pull request provides support for building with [Bazel](https://bazel.build/).

### CODE REVIEW

**Comment by pH14** [comment]
neat, thanks @bowlofstew -- I haven't used Bazel before, but I'll try this out in the coming days

**Comment by bowlofstew** [comment]
Cool, let me know if you have any questions or run into anything.  I gained some usage out of this work while integrating it with my builds here.

File: README.md [hunk 1]
```diff
@@ -50,6 +50,12 @@ TODO:
 - [ ] `setACL`
 
 
+### Building with Bazel
+
+* Compiling: `bazel build //:zdb_zk`
```
Comment (pH14) [inline README.md]: one little nit: `fdb_zk` :)

Comment (bowlofstew) [reply README.md]: doah!  Sorry, on it!  Also sorry on the delay, I have been traveling.

**Pull request merged**